recipes/itk/all/conanfile.py

```diff
@@ -0,0 +1,210 @@
+        cmake.definitions["Module_ITKImageFusion"] = "ON"
+        cmake.definitions["Module_ITKImageGradient"] = "ON"
+        cmake.definitions["Module_ITKImageGrid"] = "ON"
+        cmake.definitions["Module_ITKImageIntensity"] = "ON"
+        cmake.definitions["Module_ITKImageLabel"] = "ON"
+        cmake.definitions["Module_ITKImageSources"] = "ON"
+        cmake.definitions["Module_ITKImageStatistics"] = "ON"
+        cmake.definitions["Module_ITKIntegratedTest"] = "ON"
+        cmake.definitions["Module_ITKKLMRegionGrowing"] = "ON"
+        cmake.definitions["Module_ITKLabelMap"] = "ON"
+        cmake.definitions["Module_ITKLabelVoting"] = "ON"
+        cmake.definitions["Module_ITKLevelSets"] = "ON"
+        cmake.definitions["Module_ITKLevelSetsv4"] = "ON"
+        cmake.definitions["Module_ITKMarkovRandomFieldsClassifiers"] = "ON"
+        cmake.definitions["Module_ITKMathematicalMorphology"] = "ON"
+        cmake.definitions["Module_ITKMetricsv4"] = "ON"
+        cmake.definitions["Module_ITKNarrowBand"] = "ON"
+        cmake.definitions["Module_ITKNeuralNetworks"] = "ON"
+        cmake.definitions["Module_ITKOptimizers"] = "ON"
+        cmake.definitions["Module_ITKOptimizersv4"] = "ON"
+        cmake.definitions["Module_ITKPDEDeformableRegistration"] = "ON"
+        cmake.definitions["Module_ITKPath"] = "ON"
+        cmake.definitions["Module_ITKPolynomials"] = "ON"
+        cmake.definitions["Module_ITKQuadEdgeMeshFiltering"] = "ON"
+        cmake.definitions["Module_ITKRegionGrowing"] = "ON"
+        cmake.definitions["Module_ITKRegistrationCommon"] = "ON"
+        cmake.definitions["Module_ITKRegistrationMethodsv4"] = "ON"
+        cmake.definitions["Module_ITKReview"] = "ON"
+        cmake.definitions["Module_ITKSignedDistanceFunction"] = "ON"
+        cmake.definitions["Module_ITKSmoothing"] = "ON"
+        cmake.definitions["Module_ITKSpatialFunction"] = "ON"
+        cmake.definitions["Module_ITKThresholding"] = "ON"
+        cmake.definitions["Module_ITKVideoCore"] = "ON"
+        cmake.definitions["Module_ITKVideoFiltering"] = "ON"
+        cmake.definitions["Module_ITKVideoIO"] = "OFF"
+        cmake.definitions["Module_ITKVoronoi"] = "ON"
+        cmake.definitions["Module_ITKWatersheds"] = "ON"
+        cmake.definitions["Module_ITKDICOMParser"] = "ON"
+
+        cmake.definitions["Module_ITKVTK"] = "OFF"
+        cmake.definitions["Module_ITKVtkGlue"] = "OFF"
+
+        cmake.definitions["USE_FFTWF"] = "OFF"
+        cmake.definitions["USE_FFTWD"] = "OFF"
+
+        # Disabled on Linux (link errors)
+        cmake.definitions["Module_ITKLevelSetsv4Visualization"] = "OFF"
+
+        # Disabled because Vxl vidl is not build anymore
+        cmake.definitions["Module_ITKVideoBridgeVXL"] = "OFF"
+
+        cmake.configure(build_folder=self._build_subfolder)
+        cmake.build()
+
+    def package(self):
+        self.copy("LICENSE", dst="licenses", src=self._source_subfolder)
+        cmake = CMake(self)
```

Comment:
We prefer to call here `cmake.configure` again (probably it is not needed, but let's make all recipes as similar as possible). There is a common pattern you can check and copy from many, many recipes with the `_configure_cmake` function, i.e: https://github.com/conan-io/conan-center-index/blob/f9900cac0490539cb53faceca64b9f1f488aad79/recipes/zstd/all/conanfile.py